Readme.md

```diff
@@ -1,316 +1,18 @@
-
-### `clock.runToFrame()`
-
-Advances the clock to the next frame, firing all scheduled animation frame callbacks,
-if any, for that frame as well as any other timers scheduled along the way.
-
-### `clock.runToLast()`
-
-This takes note of the last scheduled timer when it is run, and advances the
-clock to that time firing callbacks as necessary.
-
-If new timers are added while it is executing they will be run only if they
-would occur before this time.
-
-This is useful when you want to run a test to completion, but the test recursively
-sets timers that would cause `runAll` to trigger an infinite loop warning.
-
-### `clock.setSystemTime([now])`
-
-This simulates a user changing the system clock while your program is running.
-It affects the current time but it does not in itself cause e.g. timers to fire;
-they will fire exactly as they would have done without the call to
-setSystemTime().
-
-### `clock.uninstall()`
-
-Restores the original methods on the `target` that was passed to
-`lolex.install`, or the native timers if no `target` was given.
-
-### `Date`
-
-Implements the `Date` object but using the clock to provide the correct time.
-
-### `Performance`
-
-Implements the `now` method of the [`Performance`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) object but using the clock to provide the correct time. Only available in environments that support the Performance object (browsers mostly).
-
-### `lolex.withGlobal`
-
-In order to support creating clocks based on separate or sandboxed environments (such as JSDOM), Lolex exports a factory method which takes single argument `global`, which it inspects to figure out what to mock and what features to support. When invoking this function with a global, you will get back an object with `timers`, `createClock` and `install` - same as the regular Lolex exports only based on the passed in global instead of the global environment.
-
-## Running tests
-
-Lolex has a comprehensive test suite. If you're thinking of contributing bug
-fixes or suggesting new features, you need to make sure you have not broken any
-tests. You are also expected to add tests for any new behavior.
-
-### On node:
-
-```sh
-npm test
-```
+wait.
 
-Or, if you prefer more verbose output:
+* Implements global functions like `setTimeout`, `setInterval` and so on
+* Provides a `Date` object
 
-```
-$(npm bin)/mocha ./test/lolex-test.js
-```
+In addition in browser environment lolex provides a `performance` implementation that gets its time from the clock. In Node.js environments lolex provides a `nextTick` implementation that is synchronized with the clock - and a `process.hrtime` shim that works with the clock.
 
-### In the browser
 
-[Mochify](https://github.com/mantoni/mochify.js) is used to run the tests in
-PhantomJS. Make sure you have `phantomjs` installed. Then:
+## Documentation
 
-```sh
-npm test-headless
-```
+https://sinonjs.github.io/lolex/
```

Comment:
Does this get built automatically when this PR lands? I thought for starters we could just link to the markdown file until it is up?

Reply:
I think the URL is automatically setuo when we activate github pages for the repo, but I'm really not sure. Maybe @mroderick can give some advice here? We could also link this to the bare markdown file, of course :+1:

Reply:
The url is automagically created when github pages are activated for the repository

Reply:
Great! I think we should enable github pages first before we land this then. What is the process around this?

Reply:
@benjamingr
Yes! AFAIK, we need to enable this in the settings and then have to place a `index.html` file which is taken as the landing page. As of `referee` for example, there is a decent branch which is used to publish this files https://github.com/sinonjs/referee/tree/gh-pages I guess this practice is quite common. I have no rights and clue on how to properly set this up, maybe @mroderick can help out here?

Reply:
@benjamingr 
Seems to be quite easy with the pages, there is a [command](https://www.mkdocs.org/user-guide/deploying-your-docs/) for this in `MkDocs`. I tried it and worked fine for me. See: https://mgred.github.io/lolex/

Reply:
@fatso83 @mroderick can we turn this on before the PR lands and see if it works?

Reply:
I am not currently able to turn on Pages, as the middle option that uses `/docs` is not selectable, in either Safari 12.0 or Chrome 70.

I've reported the bug to GitHub, and will turn it on once they get back to me to say that it's fixed.

Damn those custom dropdowns, if only someone had been saying all along to not re-implement the native controls ... 

Here's something to convince everyone to stop using dropdowns everywhere: https://www.youtube.com/watch?v=hcYAHix-riY

Reply:
@mroderick I think the option using the `/docs` folder is not right here, since the `lolex` documentation is built with `MkDocs`. By default Pages seem to be built using `jekyll` as described [here](https://blog.github.com/2016-08-22-publish-your-project-documentation-with-github-pages/), so this page may not be built as expected.

**Note:** We started using `jekyll` in #226 to get towards sinonjs/sinon#1651.

Reply:
The reason why I can't activate Pages using the `/docs` option, is because `/docs` doesn't exist in the `master` branch.

I think we should just merge this, and fix whatever issues we find